packages/account/src/Constants/routes.js

```diff
@@ -0,0 +1,40 @@
+const routes = {
```

Comment:
I think this was added before move to Shared was merged, so we can just that then.